src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -54,17 +55,24 @@ class ResourceService(
           // make sure resource type admin is added first because the rest depends on it
           createdAdminType <- createResourceType(resourceTypeAdmin)
 
-          result <- resourceTypes.values.filterNot(_.name == SamResourceTypes.resourceTypeAdminName).toList.parTraverse { rt =>
+          // sleep added so shadow dao can catch up, remove when removing opendj
```

Comment:
Might I suggest we add a link to the Jira issue:  https://broadworkbench.atlassian.net/browse/CA-526

That way when we get around to resolving that issue, we'll leave a comment on it saying to grep the code for all references to CA-526.  We can let @marctalbott know that this is the way we should tag code we know we want to remove.